Declarations/Contracts/Constants.ts

```diff
@@ -18,4 +18,18 @@ export function domainSupportsProperties(domain: Domain): domain is ISupportProp
         domain instanceof PageViewData ||
         domain instanceof RemoteDependencyData ||
         domain instanceof RequestData;
-}
\ No newline at end of file
+}
+
+/**
+ * Subset of Connection String fields which this SDK can parse. Lower-typecased to
+ * allow for case-insensitivity across field names
+ * @interface IConnectionStringFields
+ */
+export interface IConnectionStringFields {
```

Comment:
Nit: This is more like a static type than an interface that someone is expected to conform to and extend in an OO way. Usually interfaces like these are defined in this SDK without the `I` prefix.

Reply:
Renamed to `ConnectionStringFields`